tox.ini

```diff
@@ -3,31 +3,29 @@
 
 [tox]
 envlist = py{36,37,38,39,310}-{none,fuse2,fuse3}
-
+minversion = 3.2
+requires =
+	pkgconfig
+	cython
+	wheel
+	setuptools_scm
 [testenv]
+setenv =
+   fuse2: BORG_FUSE_IMPL=llfuse
+   fuse3: BORG_FUSE_IMPL=pyfuse3
 deps =
+    fuse2: llfuse
+    fuse3: pyfuse3
     -rrequirements.d/development.txt
```

Comment:
ah, nice! i tried something similar, but didn't get it working correctly.

but this seems to work.